### PR TITLE
Change profile screen UI

### PR DESCRIPTION
**Pull request by PuBHARGAVI**
Changes made:
- The profile icon to settings icon
- Hided RevokeId, Vc Label, Unlock auth Factor and credits and legal notices option in settings screen
- Moved the Powered By BLE text from setting options to the bottom in settings screen and added english text in all languages as discussed
- Log-out to Logout in all languages
- The Sharing Vc screen labels to "Share" and "Share with Selfie" in all languages
- Default value of VC Label to "Card"
- Added translations for Home screen header title in filipino language

### CODE REVIEW

File: screens/Profile/ProfileScreen.tsx [hunk 1]
```diff
@@ -78,11 +78,13 @@ export const ProfileScreen: React.FC<MainRouteProps> = (props) => {
           onEdit={controller.UPDATE_NAME}
           Icon="user"
         />
+        {/* Intentionally hided using {display='none'} - Refer mosip/inji/issue#607 */}
```
Comment (vharsh) [inline screens/Profile/ProfileScreen.tsx:81]: `hidden`

**Review by vharsh — APPROVED** [review]
please change `hided` to `hidden`

File: screens/Profile/Revoke.tsx [hunk 2]
```diff
@@ -15,12 +15,16 @@ import { OIDcAuthenticationOverlay } from '../../components/OIDcAuthModal';
 import { useTranslation } from 'react-i18next';
 import { useRevoke } from './RevokeController';
 
+// Intentionally hided using {display:'none'} - Refer mosip/inji/issue#607
```
[the remark this answers is not in the document]
Comment (PuBHARGAVI) [reply screens/Profile/Revoke.tsx]: done

File: screens/Profile/ProfileScreen.tsx (hunk 1 above)
Comment (PuBHARGAVI) [reply screens/Profile/ProfileScreen.tsx]: done

File: screens/Profile/Credits.tsx [hunk 3]
```diff
@@ -46,7 +47,10 @@ export const Credits: React.FC<CreditsProps> = (props) => {
   };
 
   return (
-    <ListItem bottomDivider onPress={() => setIsViewing(true)}>
+    <ListItem
+      bottomDivider
+      onPress={() => setIsViewing(true)}
+      style={{ display: 'none' }}>
```
Comment (krishnakumar4a4) [inline screens/Profile/Credits.tsx:53]: This component should be removed, not just hidden

**Pull request merged**